test/export/InstanceExporter.test.ts

```diff
@@ -185,6 +185,30 @@ describe('InstanceExporter', () => {
       expect(exported.active).toEqual(true);
     });
 
+    it('should fix top level elements to an array even if constrained on the Structure Definition', () => {
+      const condition = new Profile('TestCondition');
+      condition.parent = 'Condition';
+      doc.profiles.set(condition.name, condition);
+      const conditionInstance = new Instance('Bar');
+      conditionInstance.instanceOf = 'TestCondition';
+      doc.instances.set(conditionInstance.name, conditionInstance);
+      const fixedValRule = new FixedValueRule('category');
+      const fixedFshCode = new FshCode('foo', 'http://foo.com');
+      fixedValRule.fixedValue = fixedFshCode;
+      condition.rules.push(fixedValRule);
```

Comment:
Should `condition` have a `CardRule` added to constrain `category` to `1..1` here?

Reply:
For this one, I was trying to mirror the example that Mark mentioned in the original issue. It didn't look like that example was constraining the cardinality. It only has the rule `* category = SCT#398192003`, which I think we interpret as setting the value to the code (an object). When I undid my changes to the exporter, this test failed, so it seemed valid to me, but I'd be open to adding the `CardRule`. I was just trying to keep it as similar to the example as possible.

Do you have thoughts on which way to do this? I'm open to either.

Reply:
I doubt it makes a difference, but Mark's example is for a profile whose parent is USCoreCondition.  USCoreCondition constrains `category` to `1..*`.  So you could add that constraint if you want to be as close as possible -- although I doubt it will make a difference.

Reply:
Ah good point! I just added it (locally) and it still passes and fails correctly, so I think it makes sense to add it. I can push that!